src/accountsdb/accounts_file.zig

```diff
@@ -55,6 +74,20 @@ pub const AccountInFile = struct {
         rent_epoch: Epoch,
         owner: Pubkey,
         executable: bool,
+
+        pub fn writeToBuf(self: *const AccountInfo, buf: []u8) !usize {
+            var offset: usize = 0;
+            offset += try writeIntLittleMem(self.lamports, buf[offset..]);
```

Comment:
I'd like to point out: writeIntLittleMem doesn't return any errors, the return type should be changed, and all unnecessary error union return types it requires should be removed.

Reply:
good point 

ae91d35